### PR TITLE
Faster relay times

### PR DESCRIPTION
**Pull request by andrewnguyen22**
Flush to db, specific encoding for bloom filter

### CODE REVIEW

File: app/config.go [hunk 1]
```diff
@@ -676,6 +680,22 @@ func newDefaultGenesisState() []byte {
 	})
 	res := Codec().MustMarshalJSON(accountGenesis)
 	defaultGenesis[auth.ModuleName] = res
+	// set address as application too
+	rawApps := defaultGenesis[appsTypes.ModuleName]
+	var appsGenesis appsTypes.GenesisState
+	types.ModuleCdc.MustUnmarshalJSON(rawApps, &appsGenesis)
+	appsGenesis.Applications = append(appsGenesis.Applications, appsTypes.Application{
```
Comment (luyzdeleon) [inline app/config.go:687]: Should this be in the default genesis?

File: x/pocketcore/types/codec.go [hunk 2]
```diff
@@ -23,6 +24,7 @@ func RegisterCodec(cdc *codec.Codec) {
 	cdc.RegisterInterface((*Proof)(nil), nil)
 	cdc.RegisterConcrete(RelayProof{}, "pocketcore/relay_proof", nil)
 	cdc.RegisterConcrete(ChallengeProofInvalidData{}, "pocketcore/challenge_proof_invalid_data", nil)
+	cdc.RegisterConcrete(evidence{}, "pocketcore/evidence_persisted", nil)
```
Comment (luyzdeleon) [inline x/pocketcore/types/codec.go:27]: Is this a private type?

**Review by luyzdeleon — CHANGES_REQUESTED** [review]

File: x/pocketcore/types/codec.go (hunk 2 above)
Comment (andrewnguyen22) [reply x/pocketcore/types/codec.go:27]: Yes

File: app/config.go (hunk 1 above)
Comment (andrewnguyen22) [reply app/config.go:687]: I think so, people would likely want to do exactly what I did to test for relays

**Review by luyzdeleon — APPROVED** [review]

**Pull request merged**